Gems/WhiteBox/Code/Source/Viewport/WhiteboxMeshSelectionManipulator.h

```diff
@@ -0,0 +1,10 @@
+#include "Manipulators/ManipulatorBus.h"
+
+
+namespace WhiteBox
+{
+class MeshSelectionManipulator : public AzToolsFramework::BaseManipulator, 
+    public AzToolsFramework::ManipulatorSpace{
+
+};
+}
```

Comment:
update: This can be removed too, much better to use the selection primitives provided by EditorWhiteBoxComponentMode (e.g. `FindClosestVertexIntersection`, `FindClosestEdgeIntersection` etc... you can use that returned edge to create a Manipulator when one of these is returned. This is what we do in DefaultMode right now. For this new transform mode, we would just want to create a `TranslationManipulator` that displays until you click another vertex or click off of the manipulator (similar to how the Spline works with EditorVertexSelection)

Reply:
yep, that's what I was planning to do as well.